akka-docs/src/main/paradox/testing.md

```diff
@@ -7,11 +7,34 @@ perform tests.
 
 Akka comes with a dedicated module `akka-testkit` for supporting tests.
 
-@@@ note
+## Dependencies 
 
 Be sure to add the module `akka-testkit` to your dependencies.
 
-@@@
+sbt
+:   @@@vars
+    ```
+    "com.typesafe.akka" %% "akka-testkit" % "$akka.version$"
+    ```
+    @@@
+
+Gradle
+:   @@@vars
+    ```
+    compile group: 'com.typesafe.akka', name: 'akka-testkit_$scala.binary_version$', version: '$akka.version$'
+    ```
+    @@@
+
+Maven
+:   @@@vars
+    ```
+    <dependency>
+      <groupId>com.typesafe.akka</groupId>
+      <artifactId>akka-testkit_$scala.binary_version$</artifactId>
+      <version>$akka.version$</version>
+    </dependency>
+    ```
+    @@@
```

Comment:
It could use the nicer syntax:

```
@@dependency[sbt,Maven,Gradle] {
  group="com.typesafe.akka"
  artifact="akka-http_2.12"
  version="10.0.10"
}
```

https://developer.lightbend.com/docs/paradox/latest/features/dependencies.html

plz?

Reply:
@ktoso fixed :-) cheers mate!